liota/dccs/dcc.py

```diff
@@ -31,28 +31,40 @@
 # ----------------------------------------------------------------------------#
 
 import logging
+import json
 from abc import ABCMeta, abstractmethod
 
 from liota.entities.entity import Entity
 from liota.dcc_comms.dcc_comms import DCCComms
 from liota.entities.metrics.registered_metric import RegisteredMetric
+from liota.dcc_comms.check_connection import CheckConnection
+from liota.core.offline_queue import OfflineQueue
+from liota.core.offline_database import OfflineDatabase
+from liota.lib.utilities.offline_buffering import BufferingParams
 
 log = logging.getLogger(__name__)
 
-
 class DataCenterComponent:
-
     """
     Abstract base class for all DCCs.
     """
     __metaclass__ = ABCMeta
 
     @abstractmethod
-    def __init__(self, comms):
+    def __init__(self, comms, buffering_params):
```

Comment:
Should there be a check for buffering_params or it can be optional? Like if passed its okay otherwise it will be taken as None, as initialised automatically in __init__ methods of different dccs.

Reply:
It's fine as this is an abstract class